### PR TITLE
Set up application structure for Client

### PR DESCRIPTION
**Pull request by Cma-B**
### User Story
```
As a development team
In order to be able to start adding features
We would like to have an basic application structure
```
## Changes proposed in this pull request:
* scaffold react application set up and remove unnecessary files  
## What I have learned working on this feature:
* how to make the development branch in gitHub and set it as a default

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
```
Comment (viktorsommar) [inline public/index.html]: You can probable remove line 12 since your removed the logo from your application.

File: public/robots.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
+User-agent: *
+Disallow:
```
Comment (emtalen) [inline public/robots.txt]: You can remove this file

File: src/App.jsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+
```
Comment (emtalen) [inline src/App.jsx]: Remove extra lines

File: src/App.jsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+function App() {
+  return (
+    <div className="App">
```
Comment (emtalen) [inline src/App.jsx]: why does this div need a classname?

**Pull request merged**